Dockerfile

```
@@ -0,0 +1,9 @@
+FROM ubuntu:21.04
```

создавать образ целой ОС довольно избыточно и можно обойтись только нодой
```suggestion
FROM node:16
```